app/src/main/res/values/strings.xml

```diff
@@ -1,5 +1,5 @@
 <resources>
     <string name="app_name">JavaBaseProject</string>
-    <string name="activity_main_button_text">Button</string>
-    <string name="activity_main_title_text">Parking Text</string>
+    <string name="activity_main_button_text">PRESS ME</string>
+    <string name="activity_main_title_text">- PARKING SPOTS AVAILABLE -</string>
```

Comment:
This should be "AVAILABLE PARKING SPORTS"

Reply:
done